### ISSUE TITLE
Task alert status

### ISSUE THREAD

**Issue by TurkeyMan** [jake-phy/WindowIconList#111]
In Win7+, task buttons change colour and flash when the app enters some sort of alert state and wants user attention. It would be really awesome to duplicate this functionality if there are API's available to get this information from apps.

**Comment by jake-phy** [comment]
Yes there is but I haven't figured it out yet and this is the main bug that is holding back release.

**Comment by TurkeyMan** [comment]
Cool, look forward to it!
Thanks again for your work!

**Comment by jake-phy** [comment]
Alright give the dev a whirl.  Should display the notifications in the taskbar and in the thumbnails.

**Comment by jake-phy** [comment]
Hi TurkeyMan,  have you had a chance to test this one yet?

**Comment by TurkeyMan** [comment]
I've been looking out for it, but I haven't seen it happen yet.
The thing is, since I've never had it before on linux, I'm not sure what events should trigger it to occur :)

**Comment by jake-phy** [comment]
I know, I used pidgin to test.  But you have to turn on the notifications.  (google it)

-----Original Message-----

**Comment by TurkeyMan** [comment]
I tried Pidgin; google talks about a 'libnotify' plugin, but I don't seem to have that, I have this one 'Message Notification', which I enabled and no matter how I configured it, I didn't see any sort of attention alert on the taskbar.
I saw one option that says "Set window manager 'URGENT' hint". It didn't do anything, but I suspected it would be something like that... and if not, is there anything interest you can do with that?

**Comment by jake-phy** [comment]
Well those are settings I used to test.  What theme are you using?

**Comment by jake-phy** [comment]
One more thing, can you check and make sure the option is enabled in the settings.
